addStudy.py

```diff
@@ -0,0 +1,76 @@
+# !/usr/bin/python3
+
+# addStudy.py
+
+# python addStudy.py -c rwa_configuration.yml -l
+# python addStudy.py -c rwa_configuration.yml -a "studyName"
+
+# Adding new studies to the database
+
+# Import our libraries
+import os
+import sys
+import argparse
+import yaml
+import psycopg2
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "include"))
+from include.calibrationLib import *
+from include.database import *
+
+
+# List all studies
+def get_studies(configuration):
```

Comment:
Just pass the connection string to this function unless there is a compelling reason for it to know all the details of the configuration.